sinonjs/sinon
# Use @sinonjs/eslint-plugin-no-prototype-methods

Remove the local implementation in favour of the ESLint plugin from https://github.com/sinonjs/eslint-plugin-no-prototype-methods

 #### Purpose (TL;DR) - mandatory

Use `@sinonjs/eslint-plugin-no-prototype-methods` to help reduce duplication across repositories and aid future refactoring.

 #### How to verify - mandatory
1. Check out this branch
1. `npm install`
1. Change a file in `lib/` to use a prototype method, like `Array.prototype.sort(["one", "two"]);`
1. `npm run lint-js`
1. Observe that ESLint catches the error

 #### Checklist for author
